### PR TITLE
Codebyte examples for Map in Javascript

### LINKED ISSUE #2406: [Edit] JavaScript Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/javascript/map/has
https://www.codecademy.com/resources/docs/javascript/map/size

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by carlos-ds**
### Description

Added Codebyte examples to the following term entries for Map in Javascript: `size` and `has`.
Closes #2406.

### Type of Change

Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/map/terms/has/has.md [hunk 1]
```diff
@@ -39,3 +39,18 @@ console.log(fruits.has('Bananas')); // Output: false
 ```
 
 The first `.has()` statement checks that there are `'Apples'` in `fruits`, which returns `true`. The second statement finds that there are no `'Bananas'` in `fruits` and `false` is returned.
+
+## Codebyte Example
+
+The following example checks if there is a known abbreviation for the strings `Typescript` and `Rust`.
+
+```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/has/has.md:47]:
```suggestion
```codebyte/js
```

File: content/javascript/concepts/map/terms/size/size.md [hunk 2]
```diff
@@ -35,3 +35,23 @@ const fruits = new Map([
 
 console.log(fruits.size); // Output: 2
 ```
+
+## Codebyte Example
+
+The following example maps names to the age of the corresponding person. The value of the `size` property changes after deleting a key-value pair.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/size/size.md:41]:
```suggestion
The following example uses the `size` property to confirm the number of elements in a map after deleting a key-value pair.
```

File: content/javascript/concepts/map/terms/size/size.md [hunk 3]
```diff
@@ -35,3 +35,23 @@ const fruits = new Map([
...
+
+```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/map/terms/size/size.md:43]:
```suggestion
```codebyte/js
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@carlos-ds Just added a few comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by carlos-ds** [comment]
Hi @caupolicandiaz, thanks for reviewing! I've applied the suggested changes. If there's anything else, let me know.

**Review by caupolicandiaz — APPROVED** [review]
@carlos-ds Looks good. This is ready for a second review.

**Comment by SSwiniarski** [comment]
@carlos-ds Congrats on the new edits!

Live links:
- https://www.codecademy.com/resources/docs/javascript/map/size
- https://www.codecademy.com/resources/docs/javascript/map/has

**Pull request merged**